config.yml

```
@@ -34,10 +34,14 @@ market:
   enabled: false
   auth: test
   dex_api: https://dex.binance.org/api
+  dex_quote_update_seconds: 30
   cmc_api: https://pro-api.coinmarketcap.com
   cmc_api_key:
+  cmc_quote_update_seconds: 30
+  cmc_rate_update_seconds: 30
   fixer_api: https://data.fixer.io/api
   fixer_key:
+  fixes_rate_update_seconds: 30
```

```suggestion
  fixer_rate_update_seconds: 30
```

also it should be ~1 hour? No need to update that often as rates don't change much